conf/layer.conf

```diff
@@ -8,6 +8,6 @@ BBFILE_COLLECTIONS += "synopsys-layer"
 BBFILE_PATTERN_synopsys-layer = "^${LAYERDIR}/"
 BBFILE_PRIORITY_synopsys-layer = "6"
 
-LAYERSERIES_COMPAT_synopsys-layer = "sumo thud"
+LAYERSERIES_COMPAT_synopsys-layer = "sumo thud zeus"
```

Comment:
I guess we need to start branching now as I'm pretty sure current master (especially with your recent changes) is no longer compatible with older distro versions like `sumo` & `thud`.